src/clammit/scan_interceptor.go

```diff
@@ -113,8 +115,14 @@ func (c *ScanInterceptor) respondOnVirus(w http.ResponseWriter, filename string,
 		http.Error(w, "Internal Server Error", 500)
 		return true
 	} else if hasVirus {
+		w.Header().Set("Content-Type", c.VirusResponseContentType)
 		w.WriteHeader(c.VirusStatusCode)
-		w.Write([]byte(fmt.Sprintf("File %s has a virus!", filename)))
+
+		if len(c.VirusResponseBody) > 0 {
+			w.Write([]byte(c.VirusResponseBody))
+		} else {
```

Comment:
...remove this `if` from here, given that there will always be a response body defined.